apps/condo/domains/banking/schema/BankAccount.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * Generated by `createschema banking.BankAccount 'organization:Relationship:Organization:CASCADE; tin:Text; country:Text; routingNumber:Text; number:Text; currency:Text; approvedAt?:DateTimeUtc; approvedBy?:Text; importId?:Text; territoryCode?:Text; bankName?:Text; meta?:Json; tinMeta?:Json; routingNumberMeta?:Json'`
+ */
+
+const { get } = require('lodash')
+
+const { Text, DateTimeUtc, Select } = require('@keystonejs/fields')
+const { Json } = require('@open-condo/keystone/fields')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+
+const access = require('@condo/domains/banking/access/BankAccount')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const { IMPORT_ID_FIELD, CURRENCY_CODE_FIELD } = require('@condo/domains/common/schema/fields')
+const { COUNTRIES } = require('@condo/domains/common/constants/countries')
+
+const { validateTin } = require('@condo/domains/banking/utils/validate/tin.utils')
+const { validateRoutingNumber } = require('@condo/domains/banking/utils/validate/routingNumber.utils')
+const { validateNumber } = require('@condo/domains/banking/utils/validate/number.utils')
+
+
+const BankAccount = new GQLListSchema('BankAccount', {
+    schemaDoc: 'Bank account, that will have transactions, pulled from various integrated data sources',
+    fields: {
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        tin: {
```

Comment:
What if tin of organization will be different from tin of this account?

Reply:
> this is OK, since sometimes you need to add resource organizations or child organizations

Hmm, do you mean that resource organization will not be presented in our db?

Reply:
Yep, for these cases